### ISSUE TITLE
improvment: anotation query

### ISSUE THREAD

**Issue by coutouly** [fiskaly/grafana.surrealdb#15]
hello,
I try to use surrealdb plugin to anotate.
it is visible in the dashboard but no way to write query.
is it possible to add the query textarea ?
thanks.

![image](https://github.com/fiskaly/grafana.surrealdb/assets/7043117/e35f6b7b-96a2-4b9f-8a13-1771d5947285)

**Comment by ppaulweber** [comment]
@coutouly I'll have a look ASAP to provide an annotation integration in the data source.

**Comment by coutouly** [comment]
it would be great ...

**Comment by ppaulweber** [comment]
@coutouly of course, as stated before, I'll have a look ASAP. I'll keep you updated here.
